### ISSUE TITLE
ace

### ISSUE THREAD

**Issue by san-kumar** [cdnjs/packages#331]
**Library name:** ace

**Library JSON file link:** https://cdnjs.com/libraries/ace

**Library source (NPM package or Git repo):** https://github.com/ajaxorg/ace-builds

**Files to include:** not sure

When I open this URL **https://cdnjs.com/libraries/ace** it just says `ace version not found` instead of the normal cdnjs page.

Can you please check?

**Comment by san-kumar** [comment]
Sorry I think i posted this in the wrong repo.. closing

**Comment by MattIPv4** [comment]
@tc80 Any ideas here, the page seems to load fine now (I'm guessing it previously had `version` that wasn't committed), but is defaulting to a new `test` version, which doesn't seem to exist in the source repo and shouldn't have been added as it isn't semver?

https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/ace/package.json

**Comment by tc80** [comment]
Oh my

https://github.com/cdnjs/cdnjs/commit/776ba4cd22c236abd90a4b996674fa8192312aef

I'll look into this right now 👍

**Comment by tc80** [comment]
Yeah @MattIPv4 we removed the semver check since we were sorting by time stamp instead, but looks like it would be a good idea to enforce it again.

`2020-07-07 19:11:43.000 EDT
Jul 7 23:11:43 cdnjs-bot-6 autoupdate[24275]: 2020/07/07 23:11:43 a/ace.json: found tags in git: [v1.0.0 v1.1.01 v1.1.1 v1.1.2 v1.1.3 v1.1.4 v1.1.5 v1.1.6 v1.1.7 v1.1.8 v1.1.9 v1.2.0 v1.2.1 v1.2.2 v1.2.3 v1.2.4 v1.2.5 v1.2.6 v1.2.7 v1.2.8 v1.2.9 v1.3.0 v1.3.1 v1.3.2 v1.3.3 v1.4.0 v1.4.1 v1.4.10 v1.4.11 v1.4.12 v1.4.2 v1.4.3 v1.4.4 v1.4.5 v1.4.6 v1.4.7 v1.4.8 v1.4.9 vtest]`

Seems like a test tag `vtest` was added temporarily, which caused this issue.

Thanks @san-kumar, we will fix this shortly!
